src/util.test.ts

```diff
@@ -42,6 +42,10 @@ describe('quoteAndCombine', () => {
         "'zach', 'ryan', 'lola'",
       )
     })
+
+    it('quotes a single argument which is empty string', () => {
+      expect(quoteAndCombine('')).toEqual("''");
+    })
```

Comment:
Thank you for adding this test case!  Can you please also add this additional tes to `query.test.ts`?

```ts
it('allows empty string as property value', () => {
  let q1 = Query.g.property('foo', '')
  expect(q1.toString()).toBe("g.property('foo', '')")

  let q2 = Query.g.property({bar: ''})
  expect(q2.toString()).toBe("g.property('bar', '')")
})
```

Reply:
Done